internal/resource/config/oauth/client/oauth_client_common.go

```diff
@@ -45,8 +45,9 @@ var (
 		"ping_access_logout_capable":                  types.BoolValue(false),
 		"logout_uris":                                 types.SetNull(types.StringType),
 		"pairwise_identifier_user_type":               types.BoolValue(false),
-		"logout_mode":                                 types.StringUnknown(),
-		"back_channel_logout_uri":                     types.StringUnknown(),
+		"sector_identifier_uri":                       types.StringNull(),
```

Comment:
Was this just missing before on the default?

Reply:
yes, was missing.